### DISCUSSION TITLE
Stat panel threshold color

### DISCUSSION THREAD

**Discussion by rameesponnu** [grafana/grafana#38421] [Dashboards, Panels & Visualizations]
Hello,
I got a request from customer to change threshold color of stat panel . I have created a panel for windows servers metrics  using stat , the output value will be 0 to 100 . 
Value = 0 , full green .
Value > 0 and less than 1  , yellow color will get mixed with green , gradually increase yellow color percentage based on Value. 
Value = 1 , full yellow
Value > 1 and less than 2 , red color will start to mix with yellow .

Is there any possibility to do like this in 
![Screenshot_20210822-015329_WhatsApp](https://user-images.githubusercontent.com/89352026/130363447-39c712b7-fe2e-4612-bc04-6255ca897f7c.jpg)
 stat panel .

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
